themes/jquery.org/style.css

```diff
@@ -655,7 +655,7 @@ nav#main ul {
 	}
 
 	.dark-callout-box {
-		background-image: url(images/dark-grey-tile.png);
+		background-image: url(../jquery/images/dark-grey-tile.png);
```

Comment:
The `jquery.org` theme actually has a copy of this file. If we want to use the central one instead, should we delete the now-unused file?
<img width="665" alt="Screenshot 2023-07-03 at 20 12 17" src="https://github.com/jquery/jquery-wp-content/assets/156867/7d91dc45-3e54-4efe-bf46-da39cabe2fa9">

Reply:
Okay, removed the ones for jquery.org and brand.jquery.org.